apps/meeting/webpack.config.js

```diff
@@ -2,15 +2,15 @@
 // SPDX-License-Identifier: Apache-2.0
 
 const path = require('path');
-const HtmlWebpackInlineSourcePlugin = require('html-webpack-inline-source-plugin');
+const InlineChunkHtmlPlugin = require('react-dev-utils/InlineChunkHtmlPlugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 const app = 'meeting';
 
 module.exports = {
-  mode: 'development',
+  mode: 'production',
```

Comment:
Checking if we have a reason to change this to production?

Reply:
This is existing behavior. Previously we specified this in package.json command. I just moved it inside here.